cvat/apps/dataset_manager/formats/mot.py

```diff
@@ -67,6 +67,10 @@ def _import(src_file, task_data):
         for track in tracks.values():
             # MOT annotations do not require frames to be ordered
             track.shapes.sort(key=lambda t: t.frame)
-            # Set outside=True for the last shape in a track to finish the track
-            track.shapes[-1] = track.shapes[-1]._replace(outside=True)
+            # Append a shape with outside=True to finish the track
+            last_shape = track.shapes[-1]
+            if last_shape.frame < int(task_data.meta['task']['stop_frame']):
+                track.shapes.append(last_shape._replace(outside=True,
+                    frame=last_shape.frame + task_data.frame_step)
```

Comment:
I'm not sure that here we need to add task_data.frame_step (probably +1 is correct). You can get a shape with `frame > task_data.meta['task']['stop_frame']`. @azhavoro , could you please look at the line and give your opinion?

Reply:
With +1 I'm not sure about how filtered out frames would be handled in UI, e.g. in "go to the next keyframe" button.

Reply:
@zhiltsov-max , could you please check?

Reply:
I've checked. We can't just save the next frame (which is `+1`) without breaking the logic in many places, at least on the server, so we probably should stick to `+frame_step`.